hypertrace-view-generator/build.gradle.kts

```diff
@@ -11,6 +11,15 @@ application {
   mainClassName = "org.hypertrace.core.serviceframework.PlatformServiceLauncher"
 }
 
+hypertraceDocker {
+  defaultImage {
+    javaApplication {
+      serviceName.set("all-views")
```

Comment:
If, we will set ENV `SERVICE_NAME`, it will override, right?

Reply:
this defaults the ENV, so yeah you can override it